### PR TITLE
Spotaut-13528-aks-np

### PR DESCRIPTION
**Pull request by chandra1-n**
Create Ocean Cluster using aks-node-pool api

jira: https://spotinst.atlassian.net/browse/SPOTAUT-13527

### CODE REVIEW

File: examples/service/ocean/providers/azure_np/create/main.go [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+	// Create a new cluster.
+	out, err := svc.CloudProviderAzureNP().CreateCluster(ctx, &azure_np.CreateClusterInput{
+		Cluster: &azure_np.Cluster{
+			Name:                spotinst.String("foo"),
+			ControllerClusterID: spotinst.String("foo"),
+			AKS: &azure_np.AKS{
+				ClusterName:                     spotinst.String("foo"),
+				ResourceGroupName:               spotinst.String("foo"),
+				Region:                          spotinst.String("foo"),
```
Comment (anuragsharma-123) [inline examples/service/ocean/providers/azure_np/create/main.go:39]: Lets add valid value for region

File: examples/service/ocean/providers/azure_np/create/main.go [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+				InfrastructureResourceGroupName: spotinst.String("foo"),
+			},
+			AutoScaler: &azure_np.AutoScaler{
+				IsEnabled: spotinst.Bool(true),
+				ResourceLimits: &azure_np.ResourceLimits{
+					MaxVCPU:      spotinst.Int(120),
+					MaxMemoryGib: spotinst.Int(120),
+				},
+				Down: &azure_np.Down{
+					MaxScaleDownPercentage: spotinst.Float64(30),
```
Comment (anuragsharma-123) [inline examples/service/ocean/providers/azure_np/create/main.go:49]: Lets give float value. Also test in terraform witrh float value.

**Review by anuragsharma-123 — CHANGES_REQUESTED** [review]
Please check review comments.

File: examples/service/ocean/providers/azure_np/create/main.go (hunk 1 above)
Comment (chandra1-n) [reply examples/service/ocean/providers/azure_np/create/main.go]: added

File: examples/service/ocean/providers/azure_np/create/main.go (hunk 2 above)
Comment (chandra1-n) [reply examples/service/ocean/providers/azure_np/create/main.go]: added float value

**Review by anuragsharma-123 — APPROVED** [review]

**Pull request merged**